src/client/src/Domains/Messages/api/index.tsx

```diff
@@ -0,0 +1,40 @@
+import API from 'api';
+import * as socketio_client from 'socket.io-client';
+import { IMessageSendForm } from 'Domains/Messages/MessageSendForm';
+import { INewMessageForm } from 'Domains/Messages/NewMessageForm';
+import { IUser } from 'Domains/Accounts/api';
+
+export interface IMessage {
+    message: string;
+    sender: IUser;
+    receiver: IUser;
+    date: string;
+}
+
+export interface IConversation {
+    user: IUser;
+    latestMessage: IMessage;
+}
+
+export async function sendMessage(message: IMessageSendForm, receiver?: IUser) {
+    return API.post('message/sendMessage', {
+        message: message.message,
+        receiverId: receiver?.id
+    });
+}
+
+export async function getMessages(messenger?: IUser) {
+    return API.get<{ messages: IMessage[] }>(`message/getMessages/${messenger?.id}`);
+}
+
+export async function getConversationList() {
+    return API.get<{ conversationList: IConversation[] }>('message/getConversationList');
+}
+
+export async function createMessage(newMessageForm: INewMessageForm) {
+    return API.get < { user: IUser, error: string } >(`user/get-by-email/${newMessageForm.email}`);
+}
+
+const serverPath = 'http://localhost:5000/';
```

Comment:
Where should we get serverPath? Originally I get from src/api.tsx, but it's not working after I merged with main.

Reply:
Done.